### PR TITLE
Fix #200: Simplify the eval hint for 'vagrant service-manager env command

### LINKED ISSUE #200: Simplify the eval hint for `vagrant service-manager env command`
Copied @maxandersen comment from  https://github.com/projectatomic/vagrant-service-manager/issues/181#issuecomment-212809869

one comment is that this part from env docker:

```
# run following command to configure your shell:
# eval "$(vagrant service-manager env docker)"
```

should not be unique to the docker command.

I would have any output of env just output it at the end (once).
i.e. if I just call env I get this at the end:

```
# run following command to configure your shell:
# eval "$(vagrant service-manager env)"
```

if I call env openshift I get:

```
# run following command to configure your shell:
# eval "$(vagrant service-manager env openshift)"
```

### PR DESCRIPTION
**Pull request by brgnepal**
Fix #200.

Commands output:

```
$ vagrant service-manager env                                                
# docker env:
# Set the following environment variables to enable access to the
# docker daemon running inside of the vagrant virtual machine:
export DOCKER_HOST=tcp://10.1.2.2:2376
export DOCKER_CERT_PATH=/home/budhram/redhat/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker
export DOCKER_TLS_VERIFY=1
export DOCKER_API_VERSION=1.21

# openshift env:
# You can access the OpenShift console on: https://10.1.2.2:8443/console
# To use OpenShift CLI, run: oc login https://10.1.2.2:8443

# run following command to configure your shell:
# eval "$(vagrant service-manager env)"

$ vagrant service-manager env docker
# Set the following environment variables to enable access to the
# docker daemon running inside of the vagrant virtual machine:
export DOCKER_HOST=tcp://10.1.2.2:2376
export DOCKER_CERT_PATH=/home/budhram/redhat/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker
export DOCKER_TLS_VERIFY=1
export DOCKER_API_VERSION=1.21

# run following command to configure your shell:
# eval "$(vagrant service-manager env docker)"

$ vagrant service-manager env openshift
# You can access the OpenShift console on: https://10.1.2.2:8443/console
# To use OpenShift CLI, run: oc login https://10.1.2.2:8443

# run following command to configure your shell:
# eval "$(vagrant service-manager env openshift)"

$ vagrant service-manager env --script-readable
DOCKER_HOST=tcp://10.1.2.2:2376
DOCKER_CERT_PATH='/home/budhram/redhat/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker'
DOCKER_TLS_VERIFY=1
DOCKER_API_VERSION=1.21
OPENSHIFT_URL=https://10.1.2.2:8443
OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console

$ vagrant service-manager env docker --script-readable
DOCKER_HOST=tcp://10.1.2.2:2376
DOCKER_CERT_PATH='/home/budhram/redhat/vagrant-service-manager/.vagrant/machines/default/virtualbox/docker'
DOCKER_TLS_VERIFY=1
DOCKER_API_VERSION=1.21

$ vagrant service-manager env openshift --script-readable
OPENSHIFT_URL=https://10.1.2.2:8443
OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
```

### CODE REVIEW

**Comment by navidshaikh** [comment]
@hferentschik @maxandersen @robstryker ping can you please take a look and provide feedback?

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 - Fix #184: Make env headers comments for vagrant service-manager env @bexelbie
 - Fix #135: Refactor command.rb to make commands easier to add/maintain @budhrg
 - Adds @budhrg as co-maintainer for the plugin @navidshaikh
+- Fix #200: Simplify the eval hint for `vagrant service-manager env command` @budhrg
```
Comment (navidshaikh) [inline CHANGELOG.md]: Fix #200: Simplify the eval hint for `vagrant service-manager env` command @budhrg

**Comment by maxandersen** [comment]
Looking at the output this looks great! consistent and simple.

**Comment by maxandersen** [comment]
actually just noticed that 'env openshift' does not contain anything.

Why does 

```
$ vagrant service-manager env openshift
# You can access the OpenShift console on: https://10.1.2.2:8443/console
# To use OpenShift CLI, run: oc login https://10.1.2.2:8443

# run following command to configure your shell:
# eval "$(vagrant service-manager env openshift)"
```

not contain 

```
export OPENSHIFT_URL=https://10.1.2.2:8443
export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
```

?

**Comment by bexelbie** [comment]
We need to either add these exports (which I vote for) or remove the eval statement from OpenShift.

**Comment by bexelbie** [comment]
Honestly, we keep going back and forth on this.  When we have the vars, someone says we should cut them because no one uses them.  When we don't have the vars they get suggested for inclusion.  I am usually in the no vars camp, but would, at this point, rather just have them and the English statements and move on :)

**Comment by maxandersen** [comment]
There are no tools that will use this before we actually have them. This is a chicken-egg argument :)

so @robstryker yes @hferentschik said that but I've repeatedly said that JBoss Tools uses it or will use it and I personally would use the heck out of it in bash.

**Comment by navidshaikh** [comment]
Looks good.

**Pull request merged**